tests/TestCase.php

```diff
@@ -571,7 +572,7 @@ protected function getFakeValue($attribute, $class)
             case 'taxCategoryId':
                 return $faker->randomElement(Entities\Product::allowedTaxCategories());
             case 'accountingCode':
-                return (string) $faker->numberBetween(1000, 10000);
+                return (string)$faker->numberBetween(1000, 10000);
```

Comment:
```suggestion
                return (string) $faker->numberBetween(1000, 10000);
```

please check after using autoformatting